### PR TITLE
Es6 classes

### PR DESCRIPTION
**Pull request by rachel-lyy**
es6-classes

### CODE REVIEW

File: es6-classes/shape.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const value = 'Area is ' + this.area + ' Circumference is ' + this.circumference;
+    return value;
+  }
+}
+
+const newShape = new Shape(23, 34);
+console.log(newShape.print());
+
+class Square extends Shape {
+  constructor(area, circumference, width) {
```
Comment (scrumptiousdev) [inline es6-classes/shape.js]: Per instruction:
Write its constructor to take the **square's width as a parameter** (and nothing else)

File: es6-classes/circle.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Square {
+  constructor(area, width, circumference, radius) {
```
Comment (scrumptiousdev) [inline es6-classes/circle.js]: Per instruction:
Write its constructor to take the **circle's radius as a parameter** (and nothing else)

File: es6-classes/shape.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  print() {
+    const value = 'Area is ' + this.area + ' Circumference is ' + this.circumference;
+    return value;
+  }
+}
+
+const newShape = new Shape(23, 34);
+console.log(newShape.print());
+
+class Square extends Shape {
```
Comment (scrumptiousdev) [inline es6-classes/shape.js]: Also, let's move Square over to square.js

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

**Comment by rachel-lyy** [comment]
Square
es6-classes-square

Circle
es6-classes-circle

**Review by scrumptiousdev — APPROVED** [review]
Approved! :)

**Pull request merged**